### ISSUE TITLE
Make the app multilingual

### ISSUE THREAD

**Issue by Jefferson227** [Jefferson227/busca-cinema#4]
All the content hard coded in the project was written in portuguese. I guess it would be nice to transform the application into a multilingual app.

**Comment by timtheguy** [comment]
I would be interested in helping. Are you looking to have a toggle that changes the language? Or simply localize based on user settings? I would need you to provide translations, unless you are okay with me using Google Translate.

**Comment by Jefferson227** [comment]
Hey @timtheguy thank you for being interested in contribute :)

I'm kinda new to ionic and I think it's easier to implement a toggle button rather than get the language based on localization, right? We could start off with the toggle button and in a near future we could implement it based on localization, what do you think?

I'm okay about the Google Translate, we can make some corrections along the time.

**Comment by timtheguy** [comment]
Awesome, a toggle would definitely be easy to implement. Where would you want it in the UI? 

I'm thinking the top right, as seen here:

![screen shot 2017-10-12 at 10 37 28 pm](https://user-images.githubusercontent.com/9803215/31527881-2284e3d4-af9e-11e7-82cc-8dde714a98fe.png)

**Comment by yannbf** [comment]
Being able to toggle the language is a nice feature, but setting the most suitable language in application boot is even better. This could quickly be achievable by getting the browser's language, or you could also use [this plugin](https://ionicframework.com/docs/native/globalization/).

Example:
```//app.component.ts
import { TranslateService } from '@ngx-translate/core';

constructor(private translate: TranslateService) {
    this.initTranslate();
}

initTranslate() {
    // Set the default language for translation strings, and the current language.
    this.translate.setDefaultLang('en');

    // Gets the browser language and set it as default
    if (this.translate.getBrowserLang() !== undefined) {
        this.translate.use(this.translate.getBrowserLang());
    } else {
        this.translate.use('en'); // Set your language here
    }
}

**Comment by timtheguy** [comment]
@yannbf, I agree that localization at launch would be better - what if I only speak english and the application is immediately presented to me in another language?

@Jefferson227, would you be interested in building out the application and providing multi-lingual language changes on application start?

**Comment by Jefferson227** [comment]
For me it's more intuitive the app to start up setting the language based on my location, or browser language as shown by @yannbf in his example. But also it seems necessary to provide a button giving to the user, the option to change the language. @timtheguy, for me it's nice to put it on the header as you suggested.

One of my concerns about translation is to transform all the information which is hard coded in portuguese into translation keys or something similar. I don't know very well how to do this on ionic yet. I just need some time to do a research, but meanwhile every suggestion/contribution is welcome.
